DQMOffline/Trigger/python/BPHMonitor_cfi.py

```diff
@@ -30,24 +39,71 @@
   xmax  = cms.double(15),
 )
 
+hltBPHmonitoring.histoPSet.dRPSet = cms.PSet(
+  nbins = cms.int32 (  20 ),
+  xmin  = cms.double(   0),
+  xmax  = cms.double(20),
+)
+
+hltBPHmonitoring.histoPSet.massPSet = cms.PSet(
+  nbins = cms.int32 ( 200 ),
+  xmin  = cms.double(  2),
+  xmax  = cms.double(4.),
+)
+hltBPHmonitoring.histoPSet.dcaPSet = cms.PSet(
+  nbins = cms.int32 ( 40 ),
+  xmin  = cms.double(  0),
+  xmax  = cms.double(4.),
+)
+
+hltBPHmonitoring.histoPSet.dsPSet = cms.PSet(
+  nbins = cms.int32 ( 10),
+  xmin  = cms.double(  0),
+  xmax  = cms.double(100),
+)
+
+hltBPHmonitoring.histoPSet.cosPSet = cms.PSet(
+  nbins = cms.int32 ( 20),
+  xmin  = cms.double(  -1),
+  xmax  = cms.double(1),
+)
+hltBPHmonitoring.histoPSet.probPSet = cms.PSet(
+  nbins = cms.int32 ( 10),
+  xmin  = cms.double(  0),
+  xmax  = cms.double(1),
+)
+
 hltBPHmonitoring.tracks       = cms.InputTag("generalTracks") # tracks??
 hltBPHmonitoring.offlinePVs      = cms.InputTag("offlinePrimaryVertices") # PVs
 hltBPHmonitoring.beamSpot = cms.InputTag("offlineBeamSpot") # 
 hltBPHmonitoring.muons     = cms.InputTag("muons") # 
+hltBPHmonitoring.photons     = cms.InputTag("photons") # 
+
+#hltBPHmonitoring.DMSelection_ref = cms.string("")
+#hltBPHmonitoring.muoSelection_ref = cms.string("")
+#hltBPHmonitoring.muoSelection_ = cms.string("")
+
 
 hltBPHmonitoring.numGenericTriggerEventPSet.andOr         = cms.bool( False )
 hltBPHmonitoring.numGenericTriggerEventPSet.dbLabel       = cms.string("BPHDQMTrigger") # it does not exist yet, we should consider the possibility of using the DB, but as it is now it will need a label per path !
 hltBPHmonitoring.numGenericTriggerEventPSet.andOrHlt      = cms.bool(True)# True:=OR; False:=AND
+hltBPHmonitoring.numGenericTriggerEventPSet.andOrL1      = cms.bool(True)# True:=OR; False:=AND
 hltBPHmonitoring.numGenericTriggerEventPSet.hltInputTag   = cms.InputTag( "TriggerResults::HLT" )
```

Comment:
In fact, using "TriggerResults" may not work in offline and in MC, because the DQM is run together with the RECO step.
If it runs in an EndPath, "TriggeResults" will refer to RECO, not HLT.